docs/api/websocket.md

```diff
@@ -525,3 +525,41 @@ If an error occurs, the `success` key in the `result` message will be set to `fa
    }
 }
 ```
+
+When calling a web service and in case an exception occurs, error `code` key can be one of the following:
```

Comment:
I don't think we have yet defined a limit to what value the code string can take.